Assets/General/Scripts/ScreenText/ScriptableScreenTextUI.cs

```diff
@@ -0,0 +1,164 @@
+/*****************************************************************************
+// File Name :         ScriptableScreenTextUI.cs
+// Author :            Nick Rice
+//                     
+// Creation Date :     10/21/24
+//
+// Brief Description : This script handles data containers for readable UI elements in the game
+*****************************************************************************/
+
+using System;
+using Unity.VisualScripting;
+using UnityEngine;
+
+/// <summary>
+/// The scriptable object for tutorial text and timing
+/// </summary>
+[CreateAssetMenu(menuName = "UI Data Object/TutorialData")]
+public class ScriptableTutorialUI : ScriptableObject
+{
+    [Tooltip("Tutorial text and time variables")]
+    [SerializeField]
+    private TextAndTimerData _tutorialData;
+    
+    #region Getting Total Time
+    [Tooltip("How much time the tutorial takes to display")]
+    public uint TotalTime { get; private set; }
+
+    /// <summary>
+    /// Checks to see if it's null, then performs total time operation
+    /// </summary>
+    private void Awake()
+    {
+        _tutorialData.GetText.NullIfEmpty();
+        if (IsActive())
+        {
+            TotalTime  = _tutorialData.GetTimeBeforeText + _tutorialData.GetTimeToDisplay;
+        }
+    }
+    #endregion
+    
+    /// <summary>
+    /// Checks to see if the dialogue data is not null or unused
+    /// </summary>
+    public bool IsActive()
+    {
+        bool active = false;
```

Comment:
bool naming convention starts with words such as 'is' or 'can'. Additionally active is a bit vague. Just by looking at the name you can't tell what is active. Same would apply to this function name. Also is this line even needed if we are setting bool in the next line? Heck we might not even need the next line we could probably just return "!_tutorialData.GetText.IsUnityNull() && _tutorialData.GetText != "";"